packages/flutter/lib/src/material/slider.dart

```diff
@@ -382,6 +384,14 @@ class Slider extends StatefulWidget {
   /// Ignored if this slider is created with [Slider.adaptive].
   final Color? inactiveColor;
 
+  /// The color of the thumb when [Slider.adaptive] shows a [CupertinoSlider]
+  ///
+  /// If this color is null, the [CupertinoSlider] will have a white thumb.
+  /// (like the native default iOS slider)
+  ///
+  /// Ignored if this slider is created with [Slider].
+  final Color? cupertinoThumbColor;
```

Comment:
When possible, we'd use `MaterialStateProperty<Color>`, so that the color's value could vary depending on the thumb's default/active/disabled state.  In this case, since the same widget property is being used by the cupertino library, what you're suggesting is consistent with the way we're using theme values, yes.

Reply:
OK, good.  We would also want to add the `thumbColor` property to the `Slider` constructor too, to be consistent.

Reply:
I've updated the parameter name.

@HansMuller Could you please give me some tipps for converting it to MaterialStateProperty<Color>?